cmd/cvc-operator/controller/cstorvolumedeployment.go

```diff
@@ -333,6 +377,7 @@ func getOrCreateCStorTargetDeployment(
 							WithImagePullPolicy(corev1.PullIfNotPresent).
 							WithPortsNew(getContainerPort(80)).
 							WithEnvsNew(getDeployTemplateEnvs(string(vol.UID))).
+							WithResources(getAuxResourceRequirement(policy)).
```

Comment:
I think we have to get a conclusion about whether volume-mgmt and exporter containers should have the same resources? We can take in next PR's.

Reply:
yes, we may or may not have different aux resources for sidecar containers.
currently we are using resources, similar to cast-template.